app/src/main/res/menu/chat_message_menu.xml

```diff
@@ -6,12 +6,16 @@
         android:id="@+id/action_copy_message"
         android:icon="@drawable/ic_content_copy_white_24dp"
         android:title="@string/nc_copy_message"
-        app:showAsAction="always"/>
+        app:showAsAction="always" />
 
     <item
         android:id="@+id/action_reply_to_message"
         android:icon="@drawable/ic_reply_white_24dp"
         android:title="@string/nc_reply"
-        app:showAsAction="always"
-        />
+        app:showAsAction="always" />
+
+    <item
+        android:id="@+id/action_delete_message"
```

Comment:
@tobiasKaminsky any idea why the other 2 items have an icon? these are never used?? In the on-long-click menu there are no icons...

Reply:
In Android version <6 (if I remember correctly) it is possible to show icons.

Reply:
ok. added an icon